### PR TITLE
Tony/python/lab02

### PR DESCRIPTION
**Pull request by webavant**

### CODE REVIEW

**Review by chapman311 — CHANGES_REQUESTED** [review]
First part of Lab 2 is missing.

Please revise the 'lab02.py' to include the version 1 code, above the version 2 code; with comments in the code for which is which.
OR
Preferably, there can be a separate file for each version of the lab: 'lab02_ver1.py' and 'lab02_ver2.py'

**Comment by webavant** [comment]
Ah, my bad! I made commits for each version, assuming that was how you'd prefer it.

File: code/tony/python/lab02_version1.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+nums = list()
```
Comment (chapman311) [inline code/tony/python/lab02_version1.py]: This is an empty list. It is also an interesting way to insatiate a list. Instead you can just say `nums = []`

**Review by chapman311 — DISMISSED** [review]
Version 1 is giving a ZeroDivisionError, and appears to be missing some values in the `nums` list.

File: code/tony/python/lab02_version2.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    else:
+        try:
+            user_input = int(user_input)
+        except ValueError:
+            continue
+        nums.append(user_input)
+
+if len(nums):
+    avg = sum(nums) / len(nums)
+    print(int(avg))
```
Comment (chapman311) [inline code/tony/python/lab02_version2.py]: Type-casting the `avg` to an `int` here is not producing the correct average.

File: code/tony/python/lab02_version1.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
-nums = list()
+nums = [5, 0, 8, 3, 4, 1, 6,]
 avg = sum(nums) / len(nums)
 print(int(avg))
```
Comment (chapman311) [inline code/tony/python/lab02_version1.py]: avg = 3.85714
Your code is type-casting the `avg` to an int, which rounds it to 4; producing an incorrect output.

**Review by chapman311 — CHANGES_REQUESTED** [review]
Version 1 has a minor error resulting in an incorrect output of the average when the average has decimal places. Version 2 does not have this error,

**Review by chapman311 — APPROVED** [review]
Code looks good. Thank you for making those changes.

**Pull request merged**